overcooked_ai_py/mdp/overcooked_mdp.py

```diff
@@ -1071,25 +1084,33 @@ def process_for_player(primary_agent_idx):
             # NOTE: currently not including time left or order_list in featurization
             return np.array(state_mask_stack).astype(int)
 
+        """Next few lines changed by pk, so that check_state_to_obs_to_state can be used here"""
         # NOTE: Currently not very efficient, a decent amount of computation repeated here
-        num_players = len(overcooked_state.players)
-        final_obs_for_players = tuple(process_for_player(i) for i in range(num_players))
-        return final_obs_for_players
+        final_obs_p0 = process_for_player(0)
+        final_obs_p1 = process_for_player(1)
+
+        # Check that the func to turn the obs back into a state is working:
+        # self.check_state_to_obs_to_state(final_obs_p0, player_observing=0, state=overcooked_state)
+        # self.check_state_to_obs_to_state(final_obs_p1, player_observing=1, state=overcooked_state)
+
+        return final_obs_p0, final_obs_p1
+
```

Comment:
Revert back to original code, unless the obs -> state code is cleaned up sufficiently (but this doesn't seem useful / high priority enough to actually do, given that we're not actually using it)